### PR TITLE
Velocity Continuation for Common Image Gathers with Fourier Neural Operators

### PR DESCRIPTION
**Pull request by turquoisedragon2926**
please squash and merge

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/.quarto/
```
Comment (ziyiyin97) [inline .gitignore:1]: I would take out this file

File: _quarto.yml [hunk 2]
```diff
@@ -20,6 +20,8 @@ website:
           text: "Digital twin with control"
         - file: yin2024SEG/paper.qmd
           text: "WISER: full-Waveform variational Inference via Subsurface Extensions with Refinements"
+        - file: Rex2024SEG/paper.qmd
+          text: "Velocity Continuatition for Common Image Gathers"
```
Comment (ziyiyin97) [inline _quarto.yml]: typo

File: index.qmd [hunk 3]
```diff
@@ -9,5 +9,5 @@ All submissions to the Image24 conference with additional figures, references, .
 List of abstrac:
 
 - [Digital twin with control](DigitalTwin/abstract.qmd) An uncertainty-aware digital twin for geological carbon storage
-
+- [Velocity Continuation for Common Image Gathers with Fourier Neural Operators](Rex2024SEG/paper.qmd) A framework to obtain CIGs in a fast manner
```
Comment (ziyiyin97) [inline index.qmd]: A framework to accelerate migration-velocity analysis and uncertainty quantification

**Review by ziyiyin97 — CHANGES_REQUESTED** [review]
You can merge on your own after fixing these. Good work!

File: .gitignore (hunk 1 above)
Comment (ziyiyin97) [reply .gitignore:1]: i.e. do not include `.gitignore` in this pr

Comment (turquoisedragon2926) [reply .gitignore:1]: I put ignored the compiled file directories, is there a reason you dont want `.gitignore`

**Pull request merged**